be/src/exec/vectorized/json_scanner.cpp

```diff
@@ -437,6 +437,15 @@ Status JsonReader::_read_rows(Chunk* chunk, int32_t rows_to_read, int32_t* rows_
                 _state->append_error_msg_to_file(std::string(sv.data(), sv.size()), st.to_string());
                 LOG(WARNING) << "failed to construct row: " << st;
             }
+            if (st.is_data_quality_error() || st.is_invalid_argument()) {
```

Comment:
It seems that `read_chunk()` at line 405 has handled this but dose not work. Could you please check it and retain one of them?